ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/AdaptHiveService.java

```diff
@@ -46,15 +47,19 @@ public class AdaptHiveService {
   private static final ThreadPoolExecutor executor = new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE * 2,
       QUEUE_CAPACITY, TimeUnit.SECONDS, new LinkedBlockingDeque<>(5));
 
-  public Object upgradeHiveTable(ArcticHiveCatalog ac, TableIdentifier tableIdentifier,
+  public Object upgradeHiveTable(ArcticHiveCatalog arcticHiveCatalog, TableIdentifier tableIdentifier,
                                  UpgradeHiveMeta upgradeHiveMeta) {
     LOG.info("Start to upgrade hive table to arctic" + tableIdentifier.toString());
     executor.submit(() -> {
       runningInfoCache.put(tableIdentifier, new UpgradeRunningInfo());
       try {
+        if (!UpgradeHiveTableUtil.isSupportFormat(arcticHiveCatalog.getHMSClient(), tableIdentifier)) {
```

Comment:
I recommend to put format check logic into `UpgreadeHiveTableUtil.upgradeHiveTable`.